### DISCUSSION TITLE
CCU3 Homematic with CC2531 Stick in Redmatic

### DISCUSSION THREAD

**Discussion by tromanski** [Koenkk/zigbee2mqtt#18026] [General]
Dear all,

I try to connect my CC2531 Stick to my CCU3 with Zigbee2Mqtt and I do not really know how. 
This is my config:
- RaspberryMatic: newest version
- RedMatic Version 7.2.1 with NodeRed 3.0.2

I installed Zigbee2Mqtt via NodeRed and I have the nodes. I implemented the Zigbee2MQTT Admin Panel, but I´m not able to connect anything.
I believe, that I have to make configuration according the instruction from https://www.zigbee2mqtt.io/guide/installation/01_linux.html#configuring
I know, that my stick is connected to /dev/ttyACM0. But where can I find the configuration.yaml on the CCU3? The folder /opt/zigbee2mqtt/data/ doesn´t exist.... 

Hope that anybody can help me!

Thank you in advance

**Comment by zorrocaesar** [comment]
I have the same issue. The bridge node simply stays offline.

**Comment by zorrocaesar** [comment]
Any hints here would be much appreciated.
